File: backend/SupportPlatform.Application/DTOs/TicketDto.cs
namespace SupportPlatform.Application.DTOs;

public class TicketDto
{
    public Guid Id { get; set; }
    public string Title { get; set; } = string.Empty;
    public string Description { get; set; } = string.Empty;
    public DateTime CreatedAt { get; set; }
    public string Status { get; set; } = string.Empty;
    public string Priority { get; set; } = string.Empty;
    public Guid CompanyId { get; set; }
    public Guid CreatedByUserId { get; set; }
}

File: frontend/src/types/TicketDto.ts
export interface TicketDto {
  id: string
  title: string
  description: string
  createdAt: string
  status: 'Open' | 'InProgress' | 'Closed'
  priority: 'Low' | 'Medium' | 'High'
  companyId: string
  createdByUserId: string
}
